src/entities/activity/ui/items/ActionPlan/Document.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { forwardRef, useEffect, useState, useContext, useMemo, useCallback } from 'react';
+
+import { Box } from '@mui/material';
+
+import { DocumentContext } from './DocumentContext';
+import { useAvailableBodyHeight, useAvailableBodyWidth } from './hooks';
+import { Page } from './Page';
+import { extractActivitiesPhrasalData } from './phrasalData';
+import { Phrase } from './Phrase';
+
+import { getProgressId } from '~/abstract/lib';
+import { PhrasalTemplatePhrase } from '~/entities/activity/lib';
+import { appletModel } from '~/entities/applet';
+import { SurveyContext } from '~/features/PassSurvey';
+import { useAppSelector } from '~/shared/utils';
+import measureComponentHeight from '~/shared/utils/measureComponentHeight';
+
+type DocumentProps = {
+  phrases: PhrasalTemplatePhrase[];
+  appletTitle: string;
+  phrasalTemplateCardTitle: string;
+};
+
+export const Document = forwardRef<HTMLDivElement, DocumentProps>(
+  ({ appletTitle, phrases, phrasalTemplateCardTitle }, ref) => {
+    const context = useContext(SurveyContext);
+
+    const activityProgress = useAppSelector((state) =>
+      appletModel.selectors.selectActivityProgress(
+        state,
+        getProgressId(context.activityId, context.eventId),
+      ),
+    );
+
+    const activitiesPhrasalData = useMemo(
+      () => extractActivitiesPhrasalData(activityProgress.items),
+      [activityProgress],
+    );
+
+    const noImage = phrases.filter((phrase) => !!phrase.image).length <= 0;
+    const [pages, setPages] = useState<React.ReactNode[]>([]);
+    const availableHeight = useAvailableBodyHeight(phrasalTemplateCardTitle);
+    const availableWidth: number = (useAvailableBodyWidth as () => number)();
+    const gap = 32;
+
+    const renderPages = useCallback(async () => {
```

Comment:
Something about the way the content is being laid out on the page here is causing it to be rendered differently between 478px and 479px screen width. At 478px (and below) the content is split between two pages

<img width="528" alt="image" src="https://github.com/user-attachments/assets/62d274a3-7ece-4a00-93ca-53a0a9c430dd">

And at 479px and above, it fits on one page

<img width="569" alt="image" src="https://github.com/user-attachments/assets/2e66aeec-057f-4282-b55d-cb112253f71a">

The intention is for the pages to retain their proportions so it looks the same on mobile as it does on the desktop (and so the downloaded images also look the same). I'm not sure what the fix could be, but playing around with these numbers should lead to a solution